### ISSUE TITLE
Docker images do not use recommended configuration for tomcat

### ISSUE THREAD

**Issue by misilot** [DSpace/DSpace#9394]
**Describe the bug**
The current docker images based on [tomcat9 ](https://github.com/DSpace/DSpace/blob/main/Dockerfile#L54) do not use the recommended configuration for server.xml

https://wiki.lyrasis.org/display/DSDOC7x/Installing+DSpace#InstallingDSpace-ServletEngine(ApacheTomcat9,Jetty,CauchoResinorequivalent)

**To Reproduce**
1. docker run -it dspace/dspace:dspace-7.6.1 cat /usr/local/tomcat/conf/server.xml
2. Evaluate configuration

**Expected behavior**
The container should use the recommended best practices.

**Comment by tdonohue** [comment]
This would require a volunteer & would also require closer analysis to ensure these changes *do not impact* local development.

In other words, as stated in the [docs for these images](https://github.com/DSpace/DSpace/blob/main/dspace/src/main/docker/README.md), these images **are meant for development** (primarily).  They are not currently being released for production installation...whereas the documentation is specific to production installations.

**Comment by misilot** [comment]
I kind of get that, but are these not the same files that are used to build demo and sandbox? If not would those examples be able to be made available?

**Comment by tdonohue** [comment]
@misilot : Yes, the basic images are used for demo/sandbox.  Nonetheless, these images are not meant to be 'production ready'.  They are sample images for DSpace that you can use to build your own images. But, at this time, I would not depend on them to be fully stable/secure at all times. We do not monitor their stability or security, so there is a chance they could break at times. We also do not monitor whether vulnerable libraries exist on these images.  At this time, we only monitor vulnerabilities in DSpace itself...not, for example, whether the Tomcat on this image has known vulnerabilities, etc.  That said, it could change in the future if we find additional committers/developers wanting to support "production-ready" images in this way.

Overall, I do agree with this ticket.  I just wanted to make it clear that there's not really any "active support" for Docker in production at this time... so, this task doesn't have a clear priority.
